test/rewards/AutoCompoundingRewards.ts

```diff
@@ -634,7 +639,53 @@ describe('AutoCompoundingRewards', () => {
                     });
                 });
 
-                describe('processing rewards', () => {
+                describe('the number of programs to auto-process the rewards for', () => {
+                    it('should initially be larger than zero', async () => {
+                        expect(await autoCompoundingRewards.autoProcessRewardsCount()).to.be.gt(0);
+                    });
+
+                    it('cannot be set to zero', async () => {
```

Comment:
All the `it` descriptions don't use our convention. Why is that?

Reply:
Because the description - `the number of programs to auto-process the rewards for` - is already given in the 'describe' itself.

Reply:
Let's stay aligned with the rest of the code

Reply:
updated